docs/design.md

```diff
@@ -1,96 +1,99 @@
-  - name: db_host
-    description: Database service hostname/ip
-    default: mariadb
-    type: string
-  - name: db_port
-    description: Database service port
-    type: int
-    default: 3306
-```
-Or with an empty parameters field
-```yaml
-name: example/no-parameters 
-description: Example showing no parameters
+The `apb init` will create an example APB Spec `apb.yml` File as shown below:
+```yml
+name: my-apb
+image: <docker-org>/my-apb
+description: "My New APB"
 bindable: false
 async: optional
 parameters: []
 ```
-
-> TODO: Explain specfile base64 encoding and label stamp
-
-## Specifying Configuration Parameters
-
-`parameters` section of the specfile; `ParameterObject` array
-
-**ParameterObject**
-
-Field Name | Type | Required | Default | Description
----|---|---|---|---
-name | string| yes |  | The name of the parameter.
-required| bool | no | true | Whether or not the parameter is required.  If `false` and no default is provided, will be omitted from the parameters passed to the APB.
-description | string | yes | | A human readable description of the parameter.
-type | string | yes | | Type of parameter. `bool`, `int`, `float`, `string`, are valid
-default | bool,int,float,string|  no | | An optional default value for the parameter.
-
-
-## Running Ansible Playbook Bundles
-
-APBs are containers that are run with the following command:
-
-`docker run $container_name $action $arguments`
-
-* `container_name`
-
-The name of the APB container, i.e. `apb/etherpad`
-
-* `action`
-
-One of the 4 possible actions an APB can take. At a minimum, an APB
-must implement provision and deprovision.
-
-```
-* provision
-* deprovision
-* bind
-* unbind
-```
-
-* `arguments`
-
-Arguments in the form of a JSON payload that an APB may need to perform
-a specified action. JSON is an `Arguments` object:
-
-**Arguments**
-
-Field Name | Description
----|---
-instance_id | **Required** Service instance id. UUID string
-answers | **Required.** JSON object containing key-value answers for parameter config
-cluster | **Required.** `ClusterObject` containing targetted cluster information and credentials (TBD)
-
-> TODO: ## Authentication section to better describe ClusterObject schema and what's included
-
-> TODO: ## Dependencies! How is the container graph represented in a spec file. Is it necessary?
+The spec file will need to be edited for your specific application.
+
+#### Actions
+The following are the actions for an APB. At a minimum, an APB must implement the `provision` and `deprovision` actions.
+ * provision.yml
+   * Playbook called to handle installing application to the cluster
+ * deprovision.yml
+   * Playbook called to handle uninstalling
+ * bind.yml
+   * Playbook to grant access to another service to use this service, i.e. generates credentials
+ * unbind.yml
+   * Playbook to revoke access to this service
+
+The required named playbooks correspond to methods defined by the Open Service Broker API. For example, when the
```

Comment:
include a link to the Open Service Broker API https://github.com/openservicebrokerapi/servicebroker